quickjs.h

```diff
@@ -680,13 +680,15 @@ JS_EXTERN int JS_IsArray(JSContext *ctx, JSValueConst val);
 
 JS_EXTERN JSValue JS_NewDate(JSContext *ctx, double epoch_ms);
 
+typedef struct JSInlineCache JSInlineCache;
+
 JS_EXTERN JSValue JS_GetPropertyInternal(JSContext *ctx, JSValueConst obj,
-                                         JSAtom prop, JSValueConst receiver,
-                                         JS_BOOL throw_ref_error);
+                               JSAtom prop, JSValueConst receiver,
+                               JSInlineCache *ic, JS_BOOL throw_ref_error);
```

Comment:
There seems no way for outside code to use the IC is there? Then how about creating some `JS_GetPropertyInternal2` method on the C file which does thake the IC and leave this one alone, which would internally call `JS_GetPropertyInternal2` with IC as NULL? Same for the setter.